### PR TITLE
Update Nightwave plaza

### LINKED ISSUE #3998: [Presence | Bug] Nightwave Plaza
**Describe the Bug**
Nightwave Plaza presence is not working.

**Steps to Reproduce**

1. Check if presence has been added, if not, add.
2. Access the website.

**Expected Behavior**
Lately the site is changing its design, and I think PreMiD is no longer recognizing it because of that.

**Screenshots**
image

**Environment**
- OS: Windows 10
- Browser: Opera GX
- Version: LVL 2 (core: 76.0.4017.208)
 

**Additional context**
None.

### PR DESCRIPTION
**Pull request by vahya**
Closes PreMiD/Presences#3998

nightwave

### CODE REVIEW

File: websites/N/Nightwave Plaza/dist/metadata.json [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
+  "service": "Nightwave Plaza",
   "author": {
     "name": "ry",
     "id": "421371986824921109"
   },
-  "service": "Nightwave Plaza",
+  "category": "music",
+  "logo": "https://i.imgur.com/rNVPwya.png",
+  "thumbnail": "https://i.imgur.com/qnTU97o.jpg",
+  "version": "2.0.0,
```
Comment (ririxidev) [inline websites/N/Nightwave Plaza/dist/metadata.json:11]:
```suggestion
  "version": "2.0.0",
```

**Review by rxri — CHANGES_REQUESTED** [review]
pls support `play history` & `ratings`

**Comment by vahya** [comment]
> pls support `play history` & `ratings`

Ok

File: websites/N/Nightwave Plaza/presence.ts [hunk 2]
```diff
@@ -3,37 +3,75 @@ const presence = new Presence({
...
 
-  presence.setActivity(presenceData);
+  presenceData.buttons = [
+    {
+      label: "Listen Along",
+      url: document.location.href
+    }
+  ];
+
+  if (presenceData.details === null) {
```
Comment (SlowLife1661) [inline websites/N/Nightwave Plaza/presence.ts:73]:
```suggestion
  if (!presenceData.details) {
```

**Review by Slowlife01 — DISMISSED** [review]

File: websites/N/Nightwave Plaza/presence.ts [hunk 3]
```diff
@@ -3,37 +3,75 @@ const presence = new Presence({
...
+    }
   }
 
-  presence.setActivity(presenceData);
+  presenceData.buttons = [
+    {
+      label: "Listen Along",
+      url: document.location.href
+    }
+  ];
```
Comment (ririxidev) [inline websites/N/Nightwave Plaza/presence.ts:71]:
```suggestion
```

**Review by rxri — CHANGES_REQUESTED** [review]

**Review by rxri — APPROVED** [review]

File: websites/N/Nightwave Plaza/presence.ts (hunk 3 above)
Comment (vahya) [reply websites/N/Nightwave Plaza/presence.ts]: Why remove this?

Comment (ririxidev) [reply websites/N/Nightwave Plaza/presence.ts]: It's against our guidelines

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**